documentation/Quickstart-Clear-Containers-in-a-Ubuntu-VM.md

```diff
@@ -55,17 +55,17 @@ All you need to do to use ``ciao-down`` on your machine is:
 
 
    ```
-   $ $GOPATH/bin/ciao-down prepare -vmtype clearcontainers
+   $ $GOPATH/bin/ciao-down create clearcontainers
```

Comment:
It's unfortunate that `ciao-down` doesn't provide a version command or option that we could refer to here to ensure the user is running a version of the tool that supports the new `create` comand.

I wonder if the `go get` above should be made into a `go get -u` to re-inforce that the user needs to use the latest version which provides the `create` command?